make_wininsts.bat

```diff
@@ -1,10 +1,10 @@
-SET PATH=C:\Python26;%PATH%
```

Comment:
sorry, I'm not sure I understand the question.  Are the line endings not auto-converted for you?